contracts/FeralFileAirdropV1.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.13;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "./Authorizable.sol";
+
+contract FeralFileAirdropV1 is ERC1155, Authorizable {
+    enum Type {
+        Fungible,
+        NonFungible
+    }
+
+    // Version
+    string public constant codeVersion = "FeralFileAirdropV1";
+
+    // Airdropped addresses
+    mapping(address => bool) public airdroppedAddresses;
+
+    // Token type
+    Type public tokenType;
+
+    // Burnable flag
+    bool public burnable;
+
+    // Bridgeable flag
+    bool public bridgeable;
+
+    // Contract URI
+    string public contractURI;
+
+    // Ended flag
+    bool private _ended;
+
+    constructor(
+        Type tokenType_,
+        string memory tokenURI_,
+        string memory contractURI_,
+        bool burnable_,
+        bool bridgeable_
+    ) ERC1155(tokenURI_) {
```

Comment:
Do we need to provide an updating tokenURI function to change it in case something goes wrong or needs updating?

Reply:
Do we need to provide function `uri` to return tokenURI?
I see this one on PETROLEUM contract
https://etherscan.io/address/0xedc227ad7fef4a2c6dbc2303a5732087314a4ce9#code

Reply:
That function is default by ERC1155.